src/styles/layout/index.md.njk

```diff
@@ -0,0 +1,170 @@
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">4</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">20px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">15px</td>
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">5</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">25px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">15px</td>
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">6</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">30px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">20px</td>
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">7</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">40px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">25px</td>
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">8</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">50px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">30px</td>
+    </tr>
+    <tr class="govuk-c-table__row">
+      <th class="govuk-c-table__header" scope="row">9</th>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">60px</td>
+      <td class="govuk-c-table__cell govuk-c-table__cell--numeric">40px</td>
+    </tr>
+  </tbody>
+</table>
+
+
+The spacing override classes are structured to allow you to apply any size of the scale, using margin or padding in any direction.
+
+The first part of the class name is `govuk-!-` as with all override classes. The second part signifies whether you want to add margin, `govuk-!-m` or padding, `govuk-!-p`.
+
+If you want to add the margin or padding in a particular direction you can add `l` for left, `r` for right, `t` for top, or `b` for bottom. If you don’t specify the direction, the margin or padding will be applied to all sides of the element.
+
+The last part represents the value you want to apply govuk-!-m-r9, the `r` stands for responsive and the 9 represents 9 on the spacing scale.
+
+For example:
+
+`govuk-!-m-r9` = margin: 60px (large screens) / 40px (small screens)
+
+`govuk-!-pt-r4` = padding-top: 20px (large screens) / 15px (small screens)
+
+## Width override classes
+
+If you need to constrain the width of an element independently of the grid system you can use width override classes.
+
+As with the spacing override classes the width override classes start with `govuk-!-`. The second part of the class name signifies the width on larger screen sizes, for example `govuk-!-width-one-half`  will apply a width of 50%,  `govuk-!-width-two-thirds`will apply a width of 66.66%.
```

Comment:
Likewise, can we say 'spacing-override classes' and 'width-override classes'. Also the spacing around the classes looks a bit funky in the second sentence, but that might just be how it's displaying in here.